migrations/tenant/0025-custom-metadata.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE storage.objects ADD COLUMN user_metadata jsonb NULL;
```

Comment:
auth seems to have it as `raw_user_meta_data` but personally think user_metadata reads better